docs/deployment/k8s/index.mdx

```diff
@@ -37,172 +37,123 @@ If you are running Kubernetes locally, the following steps will help you start a
 Make sure you have a Kubernetes cluster running locally. Some good options include [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), [k3s](https://k3s.io/), or [k0s](https://k0sproject.io/).
 :::
 
-#### Deploying NATS
+#### Download the operator
 
-Add the NATS Helm repo:
+Download the operator from GitHub and navigate to the `quickstart` subfolder:
 
 ```shell
-helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+git clone https://github.com/wasmCloud/wasmcloud-operator.git
 ```
-
-Install the upstream NATS Helm chart to start a cluster with the following
-`values.yaml` file:
-
-```yaml
-config:
-  cluster:
-    enabled: true
-    replicas: 3
-  leafnodes:
-    enabled: true
-  jetstream:
-    enabled: true
-    fileStore:
-      pvc:
-        size: 5Gi
-    merge:
-      domain: default
-  websocket:
-    enabled: true
-    port: 4223
-```
-
 ```shell
-helm upgrade --install -f values.yaml nats-cluster nats/nats
+cd examples/quickstart
 ```

```

Comment:
With the changes below, the `Download the operator` section is not needed.

Reply:
Removed in the latest commit!